scripts/genproto.sh

```diff
@@ -95,18 +95,34 @@ run_go_tool github.com/hexfusion/schwag -input=Documentation/dev-guide/apispec/s
 if [ "$1" != "--skip-protodoc" ]; then
   log_callout "protodoc is auto-generating grpc API reference documentation..."
 
+  # API reference
   run rm -rf Documentation/dev-guide/api_reference_v3.md
   run_go_tool go.etcd.io/protodoc --directories="api/etcdserverpb=service_message,api/mvccpb=service_message,server/lease/leasepb=service_message,api/authpb=service_message" \
-    --title="etcd API Reference" \
     --output="Documentation/dev-guide/api_reference_v3.md" \
     --message-only-from-this-file="api/etcdserverpb/rpc.proto" \
-    --disclaimer="This is a generated documentation. Please read the proto files for more." || exit 2
+    --disclaimer='''---
```

Comment:
I don't think there are triple quotes in bash support. 
One of workarounds is to use temporary variable: 

```
DISCLAIMER=$(cat <<-END
    This is line one.
    This is line two.
    This is line three.
END
)
```

Reply:
Thanks for the workaround, I'll add that in.

Reply:
Interesting, the `'''` is working with my bash (GNU bash, version 5.0.17(1)-release), but the `cat` method you shared seems to split the string into individual arguments when passed into `--disclaimer`.

Are you able to run the script as committed @ptabor?

I'm going to look closer at `shellcheck` as well (I've used it before on other projects, but don't have it set up for this project yet).